CHANGELOG.md

```diff
@@ -34,6 +34,11 @@ Remember to align the itemized text with the first line of an item within a list
   * The `native_serialization_strict_checks` parameter to
     {func}`jax.experimental.jax2tf.convert` is deprecated in favor of the
     new `native_serializaation_disabled_checks` ({jax-issue}`#16347`).
+  * Two functions in `jax.interpreters.pxla` that have been deprecated since
+    JAX version 0.4.6 have now been removed:
+    * `device_put`: use `jax.device_put` instead.
```

Comment:
Instead of just device_put, mention the full deprecated path. I know you have mentioned it above but while reading this I skipped that part and got confused 😛

Reply:
Sounds good – I structured it this way to avoid having to repeat `jax.interpreters.pxla`, but it probably makes sense just to write it directly.